generators/run/templates/run

```diff
@@ -257,29 +257,18 @@ case $run_command in
             shift
         done
 
-        node_run "${project}-build" "yarn run build"
+        node_run "${project}-build" "" yarn run build
 
         # Run watch command in the background
         if ${run_watcher}; then
             if [ -z "${detach}" ];  then trap "kill_container ${project}-watch" EXIT; fi
-            node_run "${project}-watch" "yarn run watch" "--detach"  # Run watch in the background
+            node_run "${project}-watch" "--detach" yarn run watch  # Run watch in the background
         fi
 
-        <% if (django && db) { %>
-        # Create isolated network
-        if ! docker network inspect ${network_name} &> /dev/null; then
-            docker network create ${network_name}
-        fi
-
-        # Start the database
-        if [ -z "${detach}" ]; then trap "kill_container ${project}-db" EXIT; fi
-        start_database "${project}-db"
-        <% } %>
-
         # Run the serve container, publishing the port, and detaching if required
-        <% if (django) { %>django_run "${project}-serve" "" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach}<% if (db) { %> --network ${network_name}<% } %> ${run_serve_docker_opts} ${*}"
-        <% } else if (jekyll) { %>bundler_run "${project}-serve" "jekyll serve -P ${PORT} -H 0.0.0.0" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach} ${run_serve_docker_opts} ${*}"
-        <% } else { %>node_run "${project}-serve" "yarn run serve ${*}" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach} ${run_serve_docker_opts}"
+        <% if (django) { %>django_run "${project}-serve" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach} ${run_serve_docker_opts}"
+        <% } else if (jekyll) { %>bundler_run "${project}-serve" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach}" jekyll serve -P ${PORT} -H 0.0.0.0
+        <% } else { %>node_run "${project}-serve" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach}" yarn run serve
```

Comment:
Ah. How would you feel about committing that change to this branch yourself? You're more likely to get it right.